practice-two/src/components/Information/index.tsx

```diff
@@ -0,0 +1,118 @@
+// react
+import React, { useMemo } from "react";
+
+// types
+import { StudentProfile, IStudentInformation } from "@type/studentProfile";
+
+// assets
+import { email, phone, teacher } from "@assets/icon";
+
+// css
+import "./Information.css";
+
+export interface InformationStudentProps {
+    student: StudentProfile;
+    classmates: IStudentInformation[];
+}
+
+const filterClassmates = (
+    allClassmates: IStudentInformation[],
+    currentStudent: StudentProfile
```

Comment:
why need to pass current student in here

Reply:
Because I need this function to check if the students' class is the same or not